chain/chain/src/runtime/tests.rs

```diff
@@ -65,7 +65,7 @@ struct TestEnvConfig {
     create_flat_storage: bool,
 }
 
-/// Environment to test runtime behaviour separate from Chain.
+/// Environment to test runtime behavior separate from Chain.
```

Comment:
Here and below it may depend on UK/US English. Can we choose one or allow both?

Reply:
Yes, I think there's a locale setting somewhere. I'll explore in a future PR to add support.